### DISCUSSION TITLE
Unable to create a turborepo

### DISCUSSION THREAD

**Discussion by pratik-unhale-coditas** [vercel/turborepo#4622] [Help] [answered]
I have no proxy also my internet connection works just fine.
node -v : 18.16.0 npm -v : 9.6.4
![Screenshot (112)](https://user-images.githubusercontent.com/118805513/232775099-6d17c1d7-017c-4115-9c04-71432234a051.png)

**Comment by anthonyshew** [comment, marked as answer]
This looks like an old issue. Should be fixed!

If you're still having problems, please open an issue. 🙏
